api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java

```diff
@@ -56,6 +56,13 @@ public String readMandrillApiKey() {
     return mandrillKeys.getString("api-key");
   }
 
+  @Override
+  public String getMoodleApiKey() {
+    JSONObject mandrillKeys = new JSONObject(readToString(getCredentialsBucketName(), "moodle-keys.json"));
```

Comment:
Nit: s/mandrill/moodle/

Also, for better consistency w/ the Mandrill method name, maybe readMoodleApiKey for a method name?

Reply:
I copy pasted the code <embarrassed>